frontend/src/actions/.env

```diff
@@ -0,0 +1,2 @@
+# private token 
+EVENTBRIGHT_API_TOKEN= "KIFYHFTNBKFCIFOPNESJ"
```

Comment:
it doesnt look like we are using this? If we do need it, we can have it live in `frontend/.env`